src/Runner.Worker/ExecutionContext.cs

```diff
@@ -52,6 +52,7 @@ public interface IExecutionContext : IRunnerService
         Dictionary<string, VariableValue> JobOutputs { get; }
         ActionsEnvironmentReference ActionsEnvironment { get; }
         List<ActionsStepTelemetry> ActionsStepsTelemetry { get; }
+        List<Telemetry> Telemetry { get; }
```

Comment:
Suggestions on naming are welcome - `JobTelemetry` perhaps?

Reply:
+1 on JobTelemetry, it is sent on the jobcompleted message so it makes sense